components/CDPUserDashboard.tsx

```diff
@@ -109,36 +153,45 @@ const CDPUserDashboard = (props: Props): JSX.Element => {
     // Negative sorts "reverse alphabetically" which allows newest files to appear first
     .sort((a: CDPFile, b: CDPFile) => -a.key.localeCompare(b.key))
 
+  const isSurveyResponse = (wanted: boolean) => (file: CDPFile) =>
+    file.key.includes('survey-response') === wanted
+  const surveyResponses = files.filter(isSurveyResponse(true))
+
   return (
-    <>
-      <h3>Raw Request Data Download</h3>
-      <p>
-        These zip files contain anonymized OTP requests for the date(s)
-        indicated.
-      </p>
-
-      {files?.length === 0 && (
-        <Alert variant="info">
-          <Alert.Heading>Loading...</Alert.Heading>
-        </Alert>
+    <Tabs defaultActiveKey="raw-data" fill>
+      <Tab eventKey="raw-data" title="Raw Request Data Download">
+        <p className="mt-3">
+          These zip files contain anonymized OTP requests for the date and time
+          indicated.
+        </p>
+        <FileListing
+          cdpUser={cdpUser}
+          downloadedFiles={downloadedFiles}
+          files={files.filter(isSurveyResponse(false))}
+          setCurrentlyDownloadingFile={setCurrentlyDownloadingFile}
+        />
+      </Tab>
+      {surveyResponses.length > 0 && (
+        <Tab eventKey="survey-data" title="Survey Response Download">
+          <p className="mt-3">
+            Select a date to download all survey responses from that date.
+            Individual csv files are individual user responses.
+          </p>
+          <FileListing
+            cdpUser={cdpUser}
+            downloadedFiles={downloadedFiles}
+            files={surveyResponses.sort(
+              (a: CDPFile, b: CDPFile) =>
+                // Sort by the number of slashes in the key name so that merged responses (not in subfolders)
+                // appear first.
+                (a.key.match(/\//g) || []).length >
+                (b.key.match(/\//g) || []).length
```

Comment:
I don't think compare function is supposed to return a boolean. It should be an integer according to the JS docs. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
Probably just swap the `>` with a `-` would work.

Reply:
Woah good catch thank you! Yes that was primarily luck-based. Fixed now